R/key_colnames.R

```diff
@@ -6,21 +6,19 @@
 #'
 #' @param x an object, such as an [`epi_df`]
```

Comment:
```suggestion
#' @param x a table, such as a [`'tibble`], [`epi_df`], or [`epi_archive`]
```

Reply:
I felt I had to change it the opposite direction because `epipredict` adds implementations for non-table-y things.  Side note: `"table"` is actually a base R class for (N-dimensional) contingency tables (counting appearances of possible combinations of values of N factor/integer variables).

Reply:
Just hedged a bit instead.